app/cli.php

```diff
@@ -7,209 +7,130 @@
 use Appwrite\Event\Delete;
 use Appwrite\Event\Func;
 use Appwrite\Platform\Appwrite;
-use Utopia\Cache\Adapter\Sharding;
-use Utopia\Cache\Cache;
-use Utopia\CLI\CLI;
+use Swoole\Runtime;
+use Utopia\CLI\Adapters\Swoole as SwooleCLI;
 use Utopia\CLI\Console;
-use Utopia\Config\Config;
-use Utopia\Database\Database;
-use Utopia\Database\Document;
 use Utopia\Database\Validator\Authorization;
-use Utopia\DSN\DSN;
+use Utopia\DI\Dependency;
 use Utopia\Logger\Log;
 use Utopia\Platform\Service;
-use Utopia\Pools\Group;
 use Utopia\Queue\Connection;
 use Utopia\Registry\Registry;
 use Utopia\System\System;
 
-Authorization::disable();
-
-CLI::setResource('register', fn () => $register);
-
-CLI::setResource('cache', function ($pools) {
-    $list = Config::getParam('pools-cache', []);
-    $adapters = [];
-
-    foreach ($list as $value) {
-        $adapters[] = $pools
-            ->get($value)
-            ->pop()
-            ->getResource()
-        ;
-    }
-
-    return new Cache(new Sharding($adapters));
-}, ['pools']);
-
-CLI::setResource('pools', function (Registry $register) {
-    return $register->get('pools');
-}, ['register']);
-
-CLI::setResource('dbForConsole', function ($pools, $cache) {
-    $sleep = 3;
-    $maxAttempts = 5;
-    $attempts = 0;
-    $ready = false;
-
-    do {
-        $attempts++;
-        try {
-            // Prepare database connection
-            $dbAdapter = $pools
-                ->get('console')
-                ->pop()
-                ->getResource();
-
-            $dbForConsole = new Database($dbAdapter, $cache);
-
-            $dbForConsole
-                ->setNamespace('_console')
-                ->setMetadata('host', \gethostname())
-                ->setMetadata('project', 'console');
-
-            // Ensure tables exist
-            $collections = Config::getParam('collections', [])['console'];
-            $last = \array_key_last($collections);
-
-            if (!($dbForConsole->exists($dbForConsole->getDatabase(), $last))) { /** TODO cache ready variable using registry */
-                throw new Exception('Tables not ready yet.');
-            }
+global $global, $container;
```

Comment:
```suggestion
global $registry, $container;
```
can we just use registry?